src/components/PetInfoSection.jsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import PropTypes from "prop-types";
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHeadCell,
+  TableRow
+} from "@baltimorecounty/dotgov-components";
+
+const PetInfoSection = props => {
+  const { attributes } = props;
+  const PetAttributeUnits = attributes.filter(
+    item => item.label === "Age Unit" || item.label === "Weight Unit"
+  );
+
+  const PetAttributeRows = attributes
+    .filter(
```

Comment:
i think this can be simplified a bit.

```js
var displayAttributes = ["breed", "color", ...];

attributes.filter(attribute = displayAttributes.contains(attribute.label.toLowerCase())).map(...
```

or something along those lines. That way we can easily add another display attribute without having to add another conditional statement.